libobs/obs-scene.c

```diff
@@ -51,6 +51,7 @@ static const char *obs_scene_signals[] = {
 	"void item_remove(ptr scene, ptr item)",
 	"void reorder(ptr scene)",
 	"void item_visible(ptr scene, ptr item, bool visible)",
+	"void item_locked(ptr scene, ptr item, bool locked)",
```

Comment:
I'm not entirely certain, but just in case there's any API compatibility issues, I'd make sure this signal is put after any existing signals.